static/app/utils/profiling/renderers/selectedFrameRenderer.tsx

```diff
@@ -14,27 +14,34 @@ class SelectedFrameRenderer {
   // We allow for passing of different contexts, this allows us to use a
   // single instance of the renderer to draw overlays on multiple canvases
   draw(
-    frame: Rect,
+    frames: Rect | Rect[],
```

Comment:
I would just change this to always be Rect[], these API's get annoying to deal with otherwise